maint_tools/test_docstrings.py

```diff
@@ -10,7 +10,6 @@
 # List of modules ignored when checking for numpydoc validation.
 DOCSTRING_IGNORE_LIST = [
     "AdditiveChi2Sampler",
-    "AgglomerativeClustering",
```

Comment:
I think that you also modify the `FeatureAgglomeration` so you can remove it from the list.